### DISCUSSION TITLE
Kit WiFi Mesh

### DISCUSSION THREAD

**Discussion by LoRacolo** [indomus/forum#12] [Networking]
Buongiorno a tutti.

Riporto sul nuovo forum la mia richiesta di aiuto che avevo messo sul vecchio poco prima della chiusura.
Il router che Fastweb mi ha fornito dopo il passaggio alla FTTH 1Gb non è soddisfacente e ancora non ci sono molte alternative di proprietà per sostituirlo (ho provato con un FritzBox! 7590 senza successo forse anche perché l'assistenza di Fastweb è latitante), quindi sto pensando di prendere un kit mesh. Avete esperienza / suggerimenti su quale prendere (ca120 MQ, muri piuttosto spessi, principalmente su un piano)?

Grazie a tutti per l'aiuto.

**Comment by snakuzzo** [comment]
1. Il Fritzbox è un ottimo modem e sicuramente una buona scelta a livello consumer, ma non guardarla come "la soluzione" a problemi legati al wifi.
2. Se hai il fritzbox puoi tranquillamente utilizzarlo con Fastweb e devono darti assistenza per poter fare in modo che funzioni. Se non erro devi comunicargli il mac address del tuo modem. La soluzione quindi non è lasciar perdere...basta telefonarli e sperare di parlare con un operatore con un minimo di voglia di lavorare.
3. prima di pensare ad altre soluzioni, valuta l'utilizzo del Fritzbox e vedi come va...se hai problemi anche con il Fritz puoi provare ad accoppiarlo a dei repeater di AVM

Esperienza personale; io ho linea TIM FTTC con un 7530 e funziona benone...situazione wifi solo leggermente migliore rispetto al modem TIM (scolapasta nero) che usavo in precedenza. Ho dovuto quindi associare un repeater e sfruttare la rete mesh prevista dal Fritzbox. Con un Repeater 300 andavo bene ma ripeteva solo la rete a 2.4GHz.  Ho poi usato un 1200 che ripete anche la 5GHz. Fino a prima del rilascio del firmware 7.25 avevo seri problemi con il 1200. Dopo il rilascio del firmware 7.25 semrbra andare tutto perfettamente

**Reply by andker87** [reply]
Confermo che il fritzbox con Fastweb funziona egregiamente, e aggiungo che per quel che mi riguarda (linea FTTC) non ho avuto bisogno del loro supporto per la configurazione, ma è bastata una ricerca veloce su google. Non ricordo se basta selezionare Fastweb tra gli operatori già proposti in configurazione o procedere manualmente, ma le impostazioni si trovano.
Riguardo la linea wifi confermo i problemi di repeater fritz, almeno io col 600 ho ancora problemi nonostante la release 7.25 (anche se un pochino la situazione pare migliorata). Detto questo cerca di capire se il tuo problema di wifi è legato al Fastgate di Fastweb o a eventuali interferenze, perché potresti avere gli stessi problemi nonostante una spesa non indifferente...

**Comment by LoRacolo** [comment]
Innanzitutto grazie per le risposte. 
Per la FTTC avete completamente ragione: ho usato proprio questo FritzBox senza bisogno di coinvolgere l'assistenza di Fastweb per un paio di settimane con ottimi risultati. Poi mi hanno passato alla FTTH e non c'è stato verso: ho seguito le guide ufficiali e non, clonato il MacAddress, modificato la stringa di presentazione ma non c'è stato nulla da fare. Richiedendo il passaggio al modem libero mi hanno "palleggiato" fino a che sono arrivato in prossimità della scadenza del periodo utile per il reso. Oggi ho parlato con un loro tecnico più preparato che ha convenuto con me che i problemi (mac binding bloccato a 15 dispositivi, copertura scarca) sono dovuti ad un difetto del Fastgate e domani verranno a sostituirlo, ma temo comunque che ci sarà bisogno di qualcosa di più performante, per quello avevo pensato ad un kit mesh.

**Reply by snakuzzo** [reply]
Ma quindi il Fritzbox l'hai restituito ?

**Comment by LoRacolo** [comment]
Sì, a malincuore, perché se aspettavo che Fastweb mi chiamasse non sarei più stato in tempo a farlo se poi non avesse funzionato.

**Reply by snakuzzo** [reply]
credimi...meglio così...nel caso te lo compri tu un modem (es. 7530) e risparmi pure

**Comment by LoRacolo** [comment]
Vi aggiorno sull'evoluzione, almeno vi potete fare due risate: ieri l'operatore al telefono mi ha detto di aver fatto richiesta di intervento per cambio modem, specificando il tipo di modem per non incorrere in modello non adatto per l'ONT installato: durata prevista dell'intervento 10 minuti.
Stamani è arrivato il tecnico, ha passato 1 ora a fare speedtest nonostante gli dicessi che il problema principale non era la velocità (anche se è un po' bassa per il tipo di connessione), alla fine l'ho costretto a mettermi in vivavoce col suo "tutor" ho convinto entrambi che va cambiato il modem e quando è andato a prenderlo dal furgone... il modem che ha portato non era compatibile con la linea (di nuovo).

**Reply by snakuzzo** [reply]
ma che modem era ?

**Reply by LoRacolo** [reply]
Non l'ha nemmeno portato in casa, suppongo fosse la versione che ha in ingresso direttamente la fibra, invece che ricevere l'rj45 dall'ONT.
